src/index.js

```diff
@@ -5,9 +5,9 @@ import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <React.Fragment>
     <App />
-  </React.StrictMode>,
+  </React.Fragment>,
```

Comment:
`</React.Fragment>` can be simplified to `</>` (if you simplify the opening tag as well)